File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int num) {
        if(num < 0) return false;

        int temp = num;
        long reverse = 0;

        while(temp>0) {
            int digit = temp % 10;
            reverse = (reverse*10) + digit;
            temp = temp / 10;
        } 
        if(num == reverse) return true;
        else return false;
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if (x<0) : return False

        temp = x
        reverse = 0

        while(temp > 0) :
            digit = temp % 10
            reverse = (reverse * 10) + digit
            temp = temp // 10

        return reverse == x
        